test/README.md

```diff
@@ -0,0 +1,39 @@
+<!-- 
+Copyright 2023 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Test
+
+The tests here are intended to test the recipes in this repo to make sure it workable and up-to-date.
+
+## Running the test locally
+
+You will need to set the project to run the test locally:
+
+```
+export PROJECT_ID=<your-project>
+```
+
+To specify the location to create the test cluster, you will need to set the environment varible, it will be `us-central1-c` by default.
+
+```
+export LOCATION=<your-project>
+```
+
+Then run to start the test:
+
+```
+make test
```

Comment:
How do you run the tests with additional logging and/or verbose logging?

Reply:
There is no verbose logging within the test now. Do we still need this?

Reply:
Yes, can we make sure this gets a follow up PR?

Reply:
Sure, will make another one to address this later